ui_components/src/village/Village.js

```diff
@@ -212,10 +213,13 @@ function Village({
             }
             {villageTab == "warTable" &&
                 <WarTable
-                warLogData={warLogData}
+                playerWarLogData={playerWarLogData}
+                warRecordData={warRecordData}
```

Comment:
Maybe same comment as the presenter here - Is "data" adding something meaningful to these names?

Reply:
Yeah representing that these values are for reference and not manipulation unless as part of a server response.